### PR TITLE
test: fix receive funds test errors

### PR DESCRIPTION
**Pull request by goga-m**
## Summary
Fix `act` errors in `NavigationBar` tests (Receive funds)
Depends on https://github.com/ArkEcosystem/desktop-wallet/pull/2772

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: src/domains/wallet/components/ReceiveFunds/__snapshots__/ReceiveFunds.test.tsx.snap [hunk 1]
```diff
@@ -257,12 +256,7 @@ exports[`ReceiveFunds should render without a wallet name 1`] = `
           </div>
           <div
             class="mt-8"
-          >
-            <img
```
Comment (faustbrian) [inline src/domains/wallet/components/ReceiveFunds/__snapshots__/ReceiveFunds.test.tsx.snap:261]: QRCode disappeared here. Shouldn't it still be present in `ReceiveFunds should render without a wallet name`?

Comment (goga-m) [reply src/domains/wallet/components/ReceiveFunds/__snapshots__/ReceiveFunds.test.tsx.snap:261]: Seems strange indeed. Let me double check on Receivefunds.test and fix it

Comment (goga-m) [reply src/domains/wallet/components/ReceiveFunds/__snapshots__/ReceiveFunds.test.tsx.snap:261]: Async rendering was not properly catched in ReceiveFunds tests. Fixed now. Thanks @faustbrian!

**Comment by faustbrian** [comment]
Has 0 modified files, closing.

**Comment by goga-m** [comment]
@faustbrian indeed. These changes are already merged in #2748 :+1:

**Pull request closed without merging**